README.md

```diff
@@ -1,12 +1,20 @@
-## Run locally
-
-1. `composer install`
-1. `cp .env.example .env` and configure `.env` file
-1. `yarn install`
-1. `yarn run dev`
-1. Optional: make the search work:
+## Local development
+
+1. Install [Valet](https://laravel.com/docs/9.x/valet)
+2. Install [sqlite](https://formulae.brew.sh/formula/sqlite)
```

Comment:
Here we should say that they need a relational database, that we support MySQL and SQLite but we recommend sqlite.

Reply:
Yep, I went with the happy path that worked for me. Sqlite is a lot easier and simpler tho - and for local development in particular is the best option (no server to run).

Reply:
You are right.